src/js/models/v2/awards/additionalDetails/BaseContractAdditionalDetails.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * BaseContractAdditionalDetails.js
+ * Created by Lizzie Salita 3/5/18
+ */
+
+const BaseContractAdditionalDetails = {
```

Comment:
Many of these fields were remapped in the original `additionalDetails.js` file, so check that file to ensure you're looking at the right response fields.